### ISSUE TITLE
apparent segfault when merging hashes

### ISSUE THREAD

**Issue by FND** [k0kubun/hamlit#101]
reduced test case:
https://gist.github.com/FND/fcc8553ca764b491b9f981cebcebce6b

```haml
:ruby
  foo = "foo"
  extras = nil

%div{ extras, foo: foo }
```

that `nil` value results in what looks like a segfault with Hamlit 2.8.0 on Ruby 2.0 and 2.3

**Comment by k0kubun** [comment]
Oh... :fearful: 

Fixed and released as v2.8.1.

**Comment by FND** [comment]
Wow, that was quick - thanks a lot!
